Azure.ResourceManager.Core.Tests/TestAssets/Identity/SystemAndUserAssignedInnerExtraField.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+  "identity": {
+    "extraId" : "test",
```

Comment:
Add an additional test case that has the "extraId" in between "principalId" and "tenantId"